demos/demo1-chat-heads/Demo.jsx

```diff
@@ -1,59 +1,222 @@
-import React from 'react';
-import {StaggeredMotion, spring, presets} from '../../src/react-motion';
-import range from 'lodash.range';
+import React from "react";
+import {
+  StaggeredMotion,
+  spring,
+  Motion,
+  presets
+} from "../../src/react-motion";
+import range from "lodash.range";
+import Slider from "./Slider";
 
-export default class Demo extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {x: 250, y: 300};
-  };
+const options = [
+  {
+    description: "Constant",
+    value: "constant"
+  },
+  {
+    description: "Linear",
+    value: "linear"
+  },
+  {
+    description: "Inverse Linear",
+    value: "invertedLinear"
+  }
+];
+
+const Demo = React.createClass({
```

Comment:
If this PR is targeted at a complete refactoring, maybe consider replacing the outdated React.createClass with a ES6 class?